handler/src/test/java/io/netty/handler/ssl/OpenSslRenegotiateTest.java

```diff
@@ -21,6 +21,8 @@
 
     @Override
     public void testRenegotiateServer() throws Throwable {
+        // BoringSSL does not support renegotiate by purpose.
+        Assume.assumeFalse("BoringSSL".equals(OpenSsl.versionString()));
```

Comment:
yeah the code is correct it just return "BoringSSL". Most likely because they not cut "real" releases.